src/yubikey.rs

```diff
@@ -376,7 +388,7 @@ impl YubiKey {
     /// Change the Personal Identification Number (PIN).
     ///
     /// The default PIN code is 123456
-    pub unsafe fn change_pin(&mut self, current_pin: &[u8], new_pin: &[u8]) -> Result<(), Error> {
+    pub fn change_pin(&mut self, current_pin: &[u8], new_pin: &[u8]) -> Result<(), Error> {
```

Comment:
Oops, this is an extraneous change I wanted to do separately, but if nobody cares I think I'll just leave it in here.